packages/workflow/src/worker-interface.ts

```diff
@@ -103,14 +103,12 @@ export async function initRuntime(options: WorkflowCreateOptionsWithSourceMap):
 
   // TODO(bergundy): check if we can use `require` with the ESM sample and make all of the runtime methods sync.
 
-  // @ts-expect-error this is a webpack alias to payloadConverterPath
```

Comment:
The prose in this comment still seems useful...

Reply:
Thanks, I'll add it back.